### PR TITLE
[doc] refine comment for mask, asr_model, convolution, ctc, encoder

### PR DESCRIPTION
**Pull request by robin1001**
Change-Id: I07495d8ea4ed9b5ba07150783d0fc65e99251653

### CODE REVIEW

File: wenet/transformer/asr_model.py [hunk 1]
```diff
@@ -336,12 +372,23 @@ def attention_rescoring(self,
                             speech_lengths: torch.Tensor,
                             beam_size: int,
                             decoding_chunk_size: int = -1) -> List[int]:
-        '''
-        param: decoding_chunk_size
+        """ Apply attention rescoring decoding, CTC prefix beam search
+            is applied first to get nbest, then we resoring the nbest on
+            attention decoder with corresponding encoder out
```
Comment (glynpu) [inline wenet/transformer/asr_model.py:377]: encoder out? or decoder out?

File: wenet/transformer/convolution.py [hunk 2]
```diff
@@ -12,20 +12,20 @@
 
 
 class ConvolutionModule(nn.Module):
-    """ConvolutionModule in Conformer model.
-    Args:
-        channels (int): The number of channels of conv layers.
-        kernel_size (int): Kernerl size of conv layers.
```
Comment (glynpu) [inline wenet/transformer/convolution.py:18]: a typo: kernerl

File: wenet/transformer/convolution.py [hunk 3]
```diff
@@ -12,20 +12,20 @@
...
                  channels: int,
                  kernel_size: int = 15,
                  activation: nn.Module = nn.ReLU(),
                  causal: bool = False,
                  bias: bool = True):
-        """Construct an ConvolutionModule object."""
+        """Construct an ConvolutionModule object.
+        Args:
+            channels (int): The number of channels of conv layers.
+            kernel_size (int): Kernerl size of conv layers.
```
Comment (glynpu) [inline wenet/transformer/convolution.py:26]: a typo: kernerl

File: wenet/transformer/asr_model.py (hunk 1 above)
Comment (robin1001) [reply wenet/transformer/asr_model.py:377]: rescoring的时候输入是nbest和encoder的输出，共同输入到decoder里面做rescoring

Comment (glynpu) [reply wenet/transformer/asr_model.py:377]: I see. agree

**Comment by glynpu** [comment]
+1

File: wenet/transformer/asr_model.py [hunk 4]
```diff
@@ -217,16 +224,22 @@ def ctc_greedy_search(self,
                           speech: torch.Tensor,
                           speech_lengths: torch.Tensor,
                           decoding_chunk_size: int = -1) -> List[List[int]]:
-        '''
-        param: speech: (batch, max_len, feat_dim)
-        param: speech_length: (batch, )
-        param: decoding_chunk_size
+
```
Comment (jschenxiaoyu) [inline wenet/transformer/asr_model.py:227]: redundant blank line

File: wenet/transformer/asr_model.py [hunk 5]
```diff
@@ -254,16 +267,24 @@ def _ctc_prefix_beam_search(
             beam_size: int,
             decoding_chunk_size: int = -1
     ) -> Tuple[List[List[int]], torch.Tensor]:
-        '''
-        param: speech: (batch, max_len, feat_dim)
-        param: speech_length: (batch, )
-        param: decoding_chunk_size
+
```
Comment (jschenxiaoyu) [inline wenet/transformer/asr_model.py:270]: redundant blank line

File: wenet/utils/mask.py [hunk 6]
```diff
@@ -51,21 +61,22 @@ def subsequent_chunk_mask(
 def add_optional_chunk_mask(xs: torch.Tensor, masks: torch.Tensor,
                             use_dynamic_chunk: bool, decoding_chunk_size: int,
                             static_chunk_size: int):
-    ''' Adding optional mask for encoder.
-    Args:
-        xs: padded input, (B, L, D), L for max length
-        mask: mask for xs, (B, 1, L)
+    ''' Apply optional mask for encoder.
```
Comment (jschenxiaoyu) [inline wenet/utils/mask.py:64]: double quotation marks

**Comment by glynpu** [comment]
+1

**Review by jschenxiaoyu — APPROVED** [review]

**Review by glynpu — APPROVED** [review]
